sdk/storage/azure-storage-blob/tests/test_container_async.py

```diff
@@ -5,64 +5,48 @@
 # Licensed under the MIT License. See License.txt in the project root for
```

Comment:
nit: (Can't leave comment on part of file not in diff) but can you remove the `# coding: utf-8` at the top? I don't think we need that anymore and I've been removing it.

Reply:
That I can do!